app/src/main/java/com/example/pygmyhippo/MainActivity.java

```diff
@@ -10,27 +16,107 @@
 import androidx.navigation.ui.AppBarConfiguration;
 import androidx.navigation.ui.NavigationUI;
 
-import com.example.pygmyhippo.databinding.ActivityMainBinding;
+import com.example.pygmyhippo.databinding.UserMainActivityNagivationBinding;
+import com.example.pygmyhippo.databinding.OrganiserMainActivityNagivationBinding;
 
-public class MainActivity extends AppCompatActivity {
+import java.util.ArrayList;
+import java.util.Arrays;
 
-    private ActivityMainBinding binding;
+/**
+ * Main Activity for our android app
+ * @author Jennifer, Griffin
+ */
+public class MainActivity extends AppCompatActivity implements OnRoleSelectedListener {
+
+    private OrganiserMainActivityNagivationBinding organiserBinder;
+    private UserMainActivityNagivationBinding userBinding;
```

Comment:
NIT: any reason why it's orgniserBindER and userBindING? if not, perhaps having consistent naming to what these are would be good

Reply:
you're right